operatorcourier/validate.py

```diff
@@ -13,6 +13,30 @@ class ValidateCmd():
     crdKey = "customResourceDefinitions"
     csvKey = "clusterServiceVersions"
     pkgsKey = "packages"
+    validationJSON = dict(
```

Comment:
This must be initialized in `__init__` because dict is mutable type, it would be shared within all instances of class.

Reply:
```python
class T:
   words = []

   def add_word(self, w):
       self.words.append(w)

   def __str__(self):
      return ' '.join(self.words)

inst1 = T()
inst1.add_word('Hi')
inst1.add_word('Hello')

print("inst1", inst1)

inst2 = T()
inst2.add_word('Hallo')
print("inst2", inst2)
```
Result:
```
inst1: Hi Hello
inst2: Hi Hello Hallo
```

Reply:
Thanks for the example, I will update it accordingly.

Reply:
Addressed in the latest commit.